### DISCUSSION TITLE
无法修改颜色

### DISCUSSION THREAD

**Discussion by Linux-qitong** [linuxdeepin/developer-center#4188] [Troubleshooting | 故障排查]
论坛坛友反馈
![202304240247028141_截图_选择区域_20230424024648](https://user-images.githubusercontent.com/130018882/233872703-ae4bf68b-3e15-4d6f-9d3c-1455a7eb52dd.png)
控制中心设置个性化主题无法切换为“浅色模式”，重启无效
https://bbs.deepin.org/phone/zh/post/255618
已艾特longlong

**Comment by Decodetalkers** [comment]
已知问题，我在看

**Comment by Decodetalkers** [comment]
好，应该是解了，等dtk推送吧，之前dconfig爆炸过，设置就不生效
